Remove `getLocalWorkerdCompatibilityDate` from workers-utils and re-export `supportedCompatibilityDate` from miniflare instead (#12387)
cloudflare/workers-sdk: .changeset/quiet-roses-help.md

---
"create-cloudflare": patch
---

> Fix the fallback compatibility date being always incorrectly used when running the CLI via `pnpm`

Reviewer: ```suggestion
Avoid resorting to the fallback compatibility date when running via `pnpm`
```
